aea/crypto/ledger_apis.py

```diff
@@ -20,23 +20,22 @@
 """Module wrapping all the public and private keys cryptography."""
 from typing import Any, Dict, Optional, Tuple, Union
 
+from cosmos_crypto import CosmosApi
+from cosmos_crypto import DEFAULT_ADDRESS as COSMOS_DEFAULT_ADDRESS
+from cosmos_crypto import DEFAULT_CHAIN_ID as COSMOS_DEFAULT_CHAIN_ID
+from cosmos_crypto import DEFAULT_CURRENCY_DENOM as COSMOS_DEFAULT_CURRENCY_DENOM
+from ethereum_crypto import DEFAULT_ADDRESS as ETHEREUM_DEFAULT_ADDRESS
+from ethereum_crypto import DEFAULT_CHAIN_ID as ETHEREUM_DEFAULT_CHAIN_ID
+from ethereum_crypto import DEFAULT_CURRENCY_DENOM as ETHEREUM_DEFAULT_CURRENCY_DENOM
+from ethereum_crypto import EthereumApi
+from fetchai_crypto import DEFAULT_ADDRESS as FETCHAI_DEFAULT_ADDRESS
+from fetchai_crypto import DEFAULT_CHAIN_ID as FETCHAI_DEFAULT_CHAIN_ID
+from fetchai_crypto import DEFAULT_CURRENCY_DENOM as FETCHAI_DEFAULT_CURRENCY_DENOM
+from fetchai_crypto import FetchAIApi
```

Comment:
I'd like to completely get rid of them here. Only use the `make_x` commands. Configs we need to discuss...